src/cache/axios.ts

```diff
@@ -44,6 +44,13 @@ export interface CacheAxiosResponse<R = any, D = any> extends AxiosResponse<R, D
    * @see https://axios-cache-interceptor.js.org/config/response-object#cached
    */
   cached: boolean;
+
+  /**
+   * A simple boolean indicating if the request returned data is from valid or stale cache.
+   * 
+   * @see https://axios-cache-interceptor.js.org/config/response-object#stale
+   */
+  stale: boolean;
```

Comment:
this type might be undefined in some areas, probably a `stale?: boolean` is a better type